test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -355,46 +592,63 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
         [Fact]
         public async Task EventQueueTimeout()
```

Comment:
Turns out this test wasn't actually working and it was hiding an issue in the EventQueue drain logic :)